### DISCUSSION TITLE
Platonic solids - simple SVG examples

### DISCUSSION THREAD

**Discussion by sta-c0000** [amandaghassaei/OrigamiSimulator#45] [Show and tell]
Find attached the five Platonic solids as SVG files for OrigamiSimulator.
I'm posting here as simple SVG examples, each with only two path elements (mountain and boundary).
Set OrigamiSimulator fold to 100% since SVGs contain correct fold angles (as opacity).
[platonic_svg.zip](https://github.com/amandaghassaei/OrigamiSimulator/files/6251254/platonic_svg.zip)

**Comment by amandaghassaei** [comment]
perfect - could I add these to the examples menu as well?  again do you have a name/website to add for attribution?
thanks!

**Comment by sta-c0000** [comment]
Feel free to do anything with it, I don't want attribution, thank you!
